src/include/error.h

```diff
@@ -48,6 +48,17 @@
 #define WT_ERR_BUSY_OK(a) WT_ERR_ERROR_OK(a, EBUSY)
 #define WT_ERR_NOTFOUND_OK(a) WT_ERR_ERROR_OK(a, WT_NOTFOUND)
 
+#define WT_ERR_TEST_KEEP(a, v) \
```

Comment:
I would suggest combining with `WT_ERR_TEST`, add an arg so we conditionally take its `else` clause. There are a couple external calls to that macro that would need to pass in the arg to retain original functionality, but reusing is desirable.

Reply:
Done.